tests/rustdoc-ui/run-as-standalone.rs

```diff
@@ -0,0 +1,17 @@
+// This test ensures that if there is only one mergeable doctest, then it is
+// instead run as part of standalone doctests.
```

Comment:
How can we tell that it is run as a standalone doctest in this rmake test? Also IMO this is an unnecessary special case. If there's only one doctest, the tiny performance benefit from running standalone isn't worth adding further complexity to the implementation.

Reply:
Fair enough!